nikola/plugins/compile/pandoc.py

```diff
@@ -49,11 +50,27 @@ def set_site(self, site):
         self.config_dependencies = [str(site.config['PANDOC_OPTIONS'])]
         super().set_site(site)
 
+    def _get_pandoc_options(self, source: str) -> List[str]:
+        config_options: Union[List[str], Dict[str, List[str]]] = self.site.config['PANDOC_OPTIONS']
+        type_ = type(config_options)
+        if type_ is list:
+            pandoc_options = config_options
+        elif type_ is dict:
+            ext = source.split('.')[-1]
```

Comment:
Consider `os.path.splitext` — with that, you can also avoid the `.{}` formatting below.

Reply:
pathlib.Path would be better but I checked it is supporting Python 3.5+. I don't think writing new code using os.path is good.

Reply:
`Path` is fine, but there’s so much `os.path` in Nikola I don’t mind adding another usage.

Reply:
Oh, sorry, I checked and pathlib actually is here in 3.4+ already. Will change soon.

Reply:
Used pathlib